.github/workflows/critical-issue.yml

```
@@ -8,10 +8,20 @@ jobs:
   slack-notify:
     name: Send slack notification if issue is labeled as critical
     runs-on: ubuntu-latest
-    if: ${{ github.event.label.name == 'severity/critical' }}
     steps:
       - id: slack-notify
         name: Send slack notification
+        if: >
+          contains(github.event.issue.labels.*.name, 'severity/critical') &&
+          (
+            contains(github.event.issue.labels.*.name, 'component/zeebe') ||
+            (
+              !contains(github.event.issue.labels.*.name, 'component/operate') &&
+              !contains(github.event.issue.labels.*.name, 'component/optimize') &&
+              !contains(github.event.issue.labels.*.name, 'component/identity') &&
+              !contains(github.event.issue.labels.*.name, 'component/tasklist')
+            )
```

Just so that we don't have to hard-code a full list of components:

```suggestion
            !contains(join(github.event.issue.labels.*.name, ", "), "component/")
```

---

I tried, but using a substring is not working. https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/expressions#contains

---

The important bit is the `join` to create a new string of all labels.

Based on https://github.com/camunda/camunda/compare/0a7156e22ad1a8d7f9b43c2b829268b0fc1cd711..ced1167fc3372734818e0ab2782dbca499ca7a0c
it looks like you only tried `startsWith(array, string)` which indeed does not work.

---

Oh sorry I missed the join function 🙇 

This was my try with the substring https://github.com/camunda/camunda/pull/20781/commits/8a6a77b301e0dc5033ca9a04ba2a9cf7bb2404b7